packages/ui-react/src/containers/AccountModal/forms/Checkout.tsx

```diff
@@ -53,7 +53,7 @@ const Checkout = () => {
     },
     onSubmitSuccess: ({ couponCode }): void => setShowCouponCodeSuccess(!!couponCode),
     onSubmitError: ({ error }) => {
-      if (error instanceof FormValidationError && error.errors.order?.includes(`not found`)) {
+      if (error instanceof FormValidationError && error.errors.order?.includes(`Order with id ${order?.id} not found`)) {
```

Comment:
Are you sure you want to make it more specific based on the nullable `order`? `not found` should also match, right?

Reply:
It was this specific before. The problem was that it also caught `couponcode not found` and then navigated away (see description).